qutip/ipynbtools.py

```diff
@@ -39,10 +39,14 @@
 import qutip
 import numpy
 import scipy
-import Cython
 import matplotlib
 import IPython
 
+try:
+    import Cython
+except ImportError:
+    pass
+
```

Comment:
```suggestion
try:
    import Cython
    _cython_available = True
except ImportError:
    _cython_available = False

```
Let's rather explicitly record whether the import of Cython succeeded. This is much more obviously correct than digging in `sys.modules` where all kinds of strange things may occur.